src/app/features/blog-posts/blog-posts.component.scss

```diff
@@ -0,0 +1,221 @@
+@use 'neu' as *;
+
+.blog-posts-container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 2rem 1rem;
+  min-height: calc(100vh - 200px);
+
+  h1 {
+    color: var(--text-primary);
+    margin: 0 0 2rem;
+    font-size: 2.5rem;
+    font-weight: 700;
+    text-align: center;
+    position: relative;
+    padding-bottom: 1rem;
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      transform: translateX(-50%);
+      width: 80px;
+      height: 4px;
+      background: linear-gradient(90deg, var(--primary-color), var(--accent-color));
+      border-radius: 2px;
+    }
+  }
+
+  // Loading State
+  .loading-container {
+    @include neu-surface();
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    min-height: 300px;
+    padding: 2rem;
+    border-radius: $neu-radius;
+    margin: 1rem 0;
+    text-align: center;
+
+    .mat-mdc-progress-spinner {
+      --mdc-circular-progress-active-indicator-color: var(--primary-color);
+    }
+
+    p {
+      margin-top: 1.5rem;
+      color: var(--text-secondary);
+      font-size: 1.1rem;
+      font-weight: 500;
+    }
+  }
+
+
+  // Error State
+  .error-container {
+    @include neu-surface();
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 3rem 2rem;
+    border-radius: $neu-radius;
+    margin: 1rem 0;
+
+    .mat-icon {
+      font-size: 4rem;
+      width: 4rem;
+      height: 4rem;
+      color: #d32f2f;
+      margin-bottom: 1.5rem;
+    }
+
+    h2 {
+      color: var(--text-primary);
+      margin: 0 0 0.5rem;
+      font-size: 1.75rem;
+    }
+
+    .error-message {
+      color: #d32f2f;
```

Comment:
In general its better to extract all colors, fonts, paddings, etc, to common styles file and then use it across the project. With that you'll have easier way to apply multiple themes and do adjustments for the CSS.